modules/schema/src/test/java/org/apache/ignite/internal/schema/ExpandableByteBufTest.java

```diff
@@ -36,12 +38,13 @@
      * AllTypesDirectOrder.
      * TODO Documentation https://issues.apache.org/jira/browse/IGNITE-15859
      */
-    @Test
-    public void allTypesDirectOrder() throws Exception {
+    @ParameterizedTest
+    @ValueSource(ints = {0, 7})
+    public void allTypesDirectOrder(int strLen) throws Exception {
```

Comment:
Could you please introduce a separate test? We need to get sure that after every write and subsequent unwrap the resulting buffer is positioned to the 0